src/pug/blog/report_21.pug

```diff
@@ -0,0 +1,47 @@
+extends _blog.pug
+
+block append subtitle
+  | Progress report #21
+
+block content
+  h2 Progress report #21
+  +news_header('8 March 2023', 'patents', '')
+
+  h3 Wireless Alpakka
+  p For the last months we have been researching (and developing) the wireless functionality on the Alpakka controller, originally we decided to continue using the #[em Raspberry Pi Pico], but moving to the #[em Pico-W] variant. Our initial work was to validate a potential BT solution, focusing on 4 key requirements:
+
+  h4 Latency
+  p Pico-W to Pico-W latency tests demostrated that, after some tuning, BT can achieve latencies down to #[em 4 milliseconds]. So in this regard BT had an acceptable performance for gaming.
```

Comment:
Maybe refer to previous progress reports?

Reply:
I would move the "consistency" (which I would rename to "Jitter") here, as it relates to the variability in latency.